package.json

```diff
@@ -7,7 +7,8 @@
     "start": "PORT=4567 http-server example & npm run watch:example & wait",
     "watch:example": "watchify -d -v -t babelify example/main.js -o example/bundle.js",
     "build": "mkdir -p dist && babel src -d dist",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "LOGGER_SILENT=1 mocha --compilers js:babel-register -r should -r babel-polyfill -r src/utils/setup-jsdom -R ${REPORTER-dot} 'src/*.spec.js'",
```

Comment:
if you really need `babel-polyfill` then you should install and save it as a dev dependency. do you really need it? usually, I don't.